src/humancellatlas/data/metadata/api.py

```diff
@@ -280,13 +280,24 @@ def contact_name(self) -> str:
         return self.name
 
 
+@dataclass(init=False)
+class Accession:
+    type: str
+    value: str
+
+    def __init__(self, type_: str, value: str) -> None:
+        self.type = type_
+        self.value = value
+
+
 @dataclass(init=False)
 class Project(Entity):
     project_short_name: str
     project_title: str
     project_description: Optional[str]  # optional up to core/project/5.2.2/project_core
     publications: Set[ProjectPublication]
     contributors: Set[ProjectContact]
+    accessions: List[Accession]
```

Comment:
Can't this be a set?

Reply:
```
Error
Traceback (most recent call last):
  File "/Users/amar/dev/hca/hca-metadata-api/test/test.py", line 775, in test_project_accessions
    bundle = Bundle(uuid, version, manifest, metadata_files)
  File "/Users/amar/dev/hca/hca-metadata-api/src/humancellatlas/data/metadata/api.py", line 1016, in __init__
    self.projects = from_json_vx(Project)
  File "/Users/amar/dev/hca/hca-metadata-api/src/humancellatlas/data/metadata/api.py", line 1014, in from_json_vx
    return from_json(core_cls, json_entities, **kwargs)
  File "/Users/amar/dev/hca/hca-metadata-api/src/humancellatlas/data/metadata/api.py", line 981, in from_json
    return {entity.document_id: entity for entity in entities}
  File "/Users/amar/dev/hca/hca-metadata-api/src/humancellatlas/data/metadata/api.py", line 981, in <dictcomp>
    return {entity.document_id: entity for entity in entities}
  File "/Users/amar/dev/hca/hca-metadata-api/src/humancellatlas/data/metadata/api.py", line 978, in <genexpr>
    core_cls.from_json(entity, manifest_entry, **kwargs)
  File "/Users/amar/dev/hca/hca-metadata-api/src/humancellatlas/data/metadata/api.py", line 119, in from_json
    return sub_cls(json, metadata_manifest_entry, **kwargs)
  File "/Users/amar/dev/hca/hca-metadata-api/src/humancellatlas/data/metadata/api.py", line 324, in __init__
    self.accessions.add(Accession(type_, value_))
TypeError: unhashable type: 'Accession'
```

used @dataclass(eq=True, frozen=True) to have the dataclass provide  a __hash__ method.